ios/app/brave_core_main.mm

```diff
@@ -243,13 +239,10 @@ - (instancetype)initWithUserAgent:(NSString*)userAgent
     web::WebUIIOSControllerFactory::RegisterFactory(
         BraveWebUIControllerFactory::GetInstance());
 
-    // Setup Component Updater
+    // TODO(darkdh): move _adblockService and _backgroundImageService to
```

Comment:
please add issue link

Reply:
added in https://github.com/brave/brave-core/pull/24953/commits/80eca9f95c2eee137bb837bd8440e247a513dc58